### PR TITLE
Tel 51 create get tenants endpoint

### PR DESCRIPTION
**Pull request by AmitkumarEvry**
Fetching list of tenants
unit test cases

### CODE REVIEW

**Comment by NagarajNidagundi** [comment]
Looks good

File: src/WWTelematics.TenantService/Application/TenantManager.cs [hunk 1]
```diff
@@ -54,28 +53,18 @@ public ApplicationLoadBalancerResponse GetEntity(Guid entityId)
...
-                return ALBResponseHelper.GetDefaultALBResponse(400);
-            }
+            this.logger.LogTrace("Requesting tenant list");
 
-            int offset = (yourEntitiesQuery.Page - 1) * yourEntitiesQuery.PageSize;
-            List<Tenant> yourEntityList = this.tenantContext.Tenants.OrderBy(p => p.Id).Skip(offset).Take(yourEntitiesQuery.PageSize).ToList();
+            List<Tenant> tenants = this.tenantContext.Tenants.Where(m => m.Active.Equals(true)).ToList();
 
-            string responseBody = JsonConvert.SerializeObject(yourEntityList);
+            string responseBody = JsonConvert.SerializeObject(tenants);
```
Comment (davidw-foxtrax) [inline src/WWTelematics.TenantService/Application/TenantManager.cs]: The tenant response should be encapsulated in an object.

{ "tenants": [] }

Comment (AmitkumarEvry) [reply src/WWTelematics.TenantService/Application/TenantManager.cs]: Changed as suggested.

**Comment by WW-CICD** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: pr-CodeBuild-WWTelematicsTenantService-CodeBuildProject
* Commit ID: 3d6049c0a8a4fb15544ec874913cc17baf8339a6
* Result: FAILED
* [Build Logs](https://tlboygrlvi.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=d79abffe-ced2-44dd-94a3-7b1d0fefb6e9%2Fbuild.log) (available for 10 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by WW-CICD** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: pr-CodeBuild-WWTelematicsTenantService-CodeBuildProject
* Commit ID: e9ecef91341580e066441431e123fe5e60af67bc
* Result: FAILED
* [Build Logs](https://tlboygrlvi.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=795240b2-1195-4bf8-9fb3-0ad3240ac507%2Fbuild.log) (available for 10 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**